nomad/nomad_node_configs/initial_server.hcl

```diff
@@ -4,6 +4,12 @@ bind_addr = "0.0.0.0"
 server {
   enabled = true
   bootstrap_expect = 1
+  preemption_config {
+    batch_scheduler_enabled    = true
+    system_scheduler_enabled   = true
+    service_scheduler_enabled  = false
```

Comment:
Having this set to false will not preempt deployment jobs. Currently higher priority train jobs will preempt only lower priority train jobs, but not deployments. Which is fine for this PR if that's the intended behavior

Reply:
Yes... that was the intended behavior.

Reply:
Ok, sounds good